### ISSUE TITLE
Is there a way to disable design tokens in addon tab?

### ISSUE THREAD

**Issue by marcoscosta90** [UX-and-I/storybook-design-token#88]
I'd like to do the same as in controls, but I haven't found a way to do it with Design Token

```
  designToken: {
    files: tokenFiles,
  },
  controls: {
    disable: true
  },
```

**Comment by lcoalves** [comment]
I have the same problem!

**Comment by Sqrrl** [comment]
Thank you @lcoalves. PR is merged and released as part of v2.1.0.
